test/spec/asyncVersioningSpec.js

```diff
@@ -55,13 +59,13 @@ describe("JSONPatchQueue instance", function () {
       ];
 
       beforeEach(function () {
-        obj = {foo: 1, baz: [{qux: 'hello'}]};
-        queue.receive(obj, versionedJSONPatch3);
+        queue.obj = {foo: 1, baz: [{qux: 'hello'}]};
```

Comment:
I think we may create an issue to improve API clarity for such case. (So to continue the discussions we had on HO,not to block this issue, but to be opened for cleaner solutions)